dev-guide/src/intro.md

```diff
@@ -0,0 +1,5 @@
+# Introduction
+
+This is the developer guide for [Prusti](https://github.com/viperproject/prusti-dev/), intended to make Prusti more approachable for new contributors.
+
```

Comment:
Perhaps highlight *contributors* and add the following sentence:

For installation instructions and a tutorial on using Prusti, please confer the [user guide](URL).